docs/edgeql/ddl/attributes.rst

```diff
@@ -38,13 +41,13 @@ has an attribute defined on it, the descendants of that schema item will
 not automatically inherit the attribute.  Normal inheritance behavior can
 be turned on by declaring the attribute with the *INHERITABLE* qualifier.
 
-:eql:synopsis:`<subdefinition>`
-    Optional sequence of subdefinitions related to the new attribute.
+The following subcommands are allowed in the
+``CREATE ABSTRACT ATTRIBUTE`` block:
 
-    The following subdefinitions are allowed in the
-    ``CREATE ABSTRACT ATTRIBUTE`` block:
-
-    * :eql:stmt:`SET ATTRIBUTE`
+:eql:synopsis:`SET ATTRIBUTE <attribute> := <value>`
+    Attributes can also have attributes. Set the *attribute* of the
+    enclosing attribute to a specific *value*. See :eql:stmt:`SET
```

Comment:
Please make sure `:eql:stmt:` fits on one line -- it's easier to read that way

Reply:
Cleaned those up.